packages/broker/src/StreamMetrics.ts

```diff
@@ -138,8 +142,10 @@ export class StreamMetrics {
                     this.report.network.connections = throttledAvg(this.report.network.connections, (metricsReport.metrics.WebRtcEndpoint.connections as any).rate || 0)
 
                     if (metricsReport.metrics['broker/cassandra']) {
-                        this.report.storage.bytesWrittenPerSec = throttledAvg(this.report.storage.bytesWrittenPerSec, (metricsReport.metrics['broker/cassandra']) ? (metricsReport.metrics['broker/cassandra'].writeBytes as any).rate: 0)
-                        this.report.storage.bytesReadPerSec = throttledAvg(this.report.storage.bytesReadPerSec, (metricsReport.metrics['broker/cassandra']) ? (metricsReport.metrics['broker/cassandra'].readBytes as any).rate : 0)
+                        // @ts-expect-error
```

Comment:
What is ts complaining about in these places? Can we fix it at the source?

Reply:
I wonder if using `?.` would make ts happy, e.g. `metricsReport.metrics['broker/cassandra']?.writeBytes?.rate || 0`

Reply:
The logic ended up being a bit more involved but the @ts-expect-error is gone anyways